backend/src/models/AbstractManager.js

```diff
@@ -14,6 +14,12 @@ class AbstractManager {
     return this.connection.query(`select * from  ${this.table}`);
   }
 
+  findSearch(research) {
+    return this.connection.query(
+      `select * from  ${this.table} WHERE name LIKE '%${research}%' OR brand LIKE '%${research}%'`
```

Comment:
congratulations, you just allowed SQL injections :p

https://www.w3schools.com/sql/sql_injection.asp

use markers `?` to inject values in SQL in a secure way